adlfs/core.py

```diff
@@ -42,21 +42,21 @@ class AzureDLFileSystem(object):
     ----------
     store : str ("")
```

Comment:
Not so - these are passed down to the REST interface and used in constructing the URL.